package.json

```diff
@@ -8,14 +8,21 @@
     "format": "yarn lint && yarn prettier --write .",
     "lint": "eslint . --fix --ext .js,.ts",
     "start": "tsc && node dist/index.js",
-    "update": "yarn upgrade-interactive --latest"
+    "update": "yarn upgrade-interactive --latest",
+    "test": "NODE_ENV=test mocha -r ts-node/register src/tests/*.ts"
   },
   "dependencies": {
     "bufferutil": "^4.0.6",
+    "chai": "^4.3.6",
```

Comment:
We should move all these packages to dev dependencies.

Reply:
I did but eslint was complaining so I had to move it here. Not sure why tho :thinking:

Reply:
I think we need to add exceptions for chai in the eslint config.